docs/tutorials/testing_with_control_center.rst

```diff
@@ -208,78 +214,93 @@ the controls for setting parameters.
+  the timeout settings on a module, refer to the :ref:`manual_timeout` section of the Feature Reference Manual.
 
 The figure below shows what your Tuning tab should look like after setting the parameters. Some of your default settings may be slightly different on a 
-2306, but that is fine:
+a module other than an 8108, but only the *Max Volts* and *Timeout* parametes are important for this example:
 
-.. figure:: ../_static/tutorial_images/control_center_tutorial/control_center_tuning.JPG
+.. figure:: ../_static/tutorial_images/control_center_tutorial/control_center_tuning.PNG
     :align: center
     :alt: Tuning Tab Settings
 
-    Tuning Tab Settings
+    Tuning Tab Settings on Vertiq 8108 with Firmware Version 0.0.7
 
-.. warning:: Double check that the motor is secured and there is no propeller attached before applying any of these testing parameters.
+.. warning:: Double check that the module is secured and there is no propeller attached before applying any of these testing parameters.
 
-Now at long last we can make the motor spin. Open the Testing tab, and scroll down to the bottom. At the bottom of the tab you should see 5 important testing parameters:
+Testing the Module
+*******************
+Now at long last you can make the module spin. Open the Testing tab, and scroll down to the bottom. At the bottom of the tab you should see 5 important testing parameters:
 
-* **Coast**: This commands the motor to coast, which means that the motor allows its rotor to slowly decelerate and come to a stop. It is a safe way 
-  to stop the motor when testing, since it does not cause the motor to rapidly decelerate.
-* **ESC Input**: This commands the motor to spin in the same way that a typical hobby protocol used by an ESC would, i.e. it sends a percentage throttle command. So setting this
-  to 0.5 will send the motor a 50% throttle command. Since we are in Voltage mode, this will be interpreted as a percentage of the maximum voltage, which should be 5V. So setting 
-  this to 0.5 should set the motor to the same speed as setting the Voltage parameter to 2.5V.
-* **PWM**: This tells the motor to spin with a percentage of its input voltage. For example, if your input voltage is 20V, and you set this parameter to 0.5, the motor will spin with
+* **Coast**: This commands the module to coast, which means that the module allows its rotor to slowly decelerate and come to a stop. It is a safe way 
+  to stop the module when testing, since it does not cause the module to rapidly decelerate. 
+* **PWM**: This tells the module to spin with a percentage of its input voltage. For example, if your input voltage is 20V, and you set this parameter to 0.5, the module will spin with
   a 10V drive voltage, since 50% of 20V is 10V. A positive PWM percentage always goes in the counter-clockwise direction, and a negative PWM percentage always goes clockwise.
```

Comment:
is this always true, or does it depend on the configuration? I know we have it configured this way for this example, but we might want to make it clear that it's not always always the case if it isn't

Reply:
That is actually always true, the clockwise/counter-clockwise setting only affects stuff that comes in through the ESC propeller input parser, this PWM command goes right past that. In the modules actual low level stuff, the positive direction is always counter-clockwise